django/conf/__init__.py

```diff
@@ -201,6 +201,14 @@ def __init__(self, settings_module):
             setattr(self, 'PASSWORD_RESET_TIMEOUT', self.PASSWORD_RESET_TIMEOUT_DAYS * 60 * 60 * 24)
             warnings.warn(PASSWORD_RESET_TIMEOUT_DAYS_DEPRECATED_MSG, RemovedInDjango40Warning)
 
+        if self.SECRET_KEY:
```

Comment:
948a874 removed the validation of the secret key here on purpose; please move it accordingly.

Reply:
Yes.. working on this.